qa/L0_iterative_tutorial/test.sh

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
nit similar to Olga's comment - can we rename this to `L0_tutorial_iterative_scheduling` (and future `L0_tutorial_*`) so we can quickly identify/group the tutorial repo based tests?